src/DependencyInjection/BaldinofRoadRunnerExtension.php

```diff
@@ -143,6 +145,14 @@ private function loadIntegrations(ContainerBuilder $container, array $config): v
                 ->addArgument(new Reference(HubInterface::class))
                 ->setAutoconfigured(true);
 
+            $container
+                ->register(SentryTracingRequestListenerDecorator::class)
+                ->setDecoratedService(TracingRequestListener::class)
+                ->setArguments([
+                    new Reference('.inner'),
```

Comment:
I think the `.inner` syntax was not supported can you try: 

```suggestion
                    new Reference(SentryTracingRequestListenerDecorator::class.'.inner'),
```

I think if it was not working before it's because it was `TracingRequestListener::class.'.inner'`